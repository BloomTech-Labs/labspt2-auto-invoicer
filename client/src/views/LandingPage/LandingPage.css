.landing_page {
  width: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100vh;
}
