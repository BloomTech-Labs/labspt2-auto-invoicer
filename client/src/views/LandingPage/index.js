import React from 'react';

// import components here
import Landing from '../../components/Landing';

// import styles here
import './LandingPage.css';

const LandingPage = props => {
  return (
    <section className="landing-page">
      <Landing
      // click={props.click}
      />
    </section>
  );
};

export default LandingPage;
