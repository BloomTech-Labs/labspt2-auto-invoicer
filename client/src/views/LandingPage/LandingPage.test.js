import React from "react";
import { render } from "react-testing-library";
import "jest-dom/extend-expect";
import LandingPage from "../LandingPage";

describe("<LandingPage />", () => {
  it("renders Landing Page view", () => {
    const { asFragment } = render(<LandingPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
