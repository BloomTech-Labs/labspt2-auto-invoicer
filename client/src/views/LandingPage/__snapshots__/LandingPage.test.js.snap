// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage /> renders Landing Page view 1`] = `
<DocumentFragment>
  <section
    class="landing-page"
  >
    <div
      class="HeadlineText-container-1"
    >
      <div
        class="HeadlineText-root-2"
      >
        <h1
          class="MuiTypography-root-5 MuiTypography-h1-17 HeadlineText-headlineText-3"
        >
          Invoicing Solved
        </h1>
        <br />
        <h4
          class="MuiTypography-root-5 MuiTypography-h4-20 HeadlineText-subtext-4"
        >
          Turn sales into cashflow.
        </h4>
      </div>
    </div>
    <div
      class="LandingCTA-container-41"
    >
      <div
        class="LandingCTA-root-42"
      >
        <h6
          class="MuiTypography-root-5 MuiTypography-subtitle1-23"
        >
          Create a free invoice on the go or sign-up and get unlimited invoices today!
        </h6>
      </div>
    </div>
    <div
      class="LandingFeaturesText-container-43"
    >
      <div
        class="LandingFeaturesText-features-44"
      >
        <div
          class="LandingFeaturesText-top-45"
        >
          <svg>
            undraw_printing_invoices_5r4r.svg
          </svg>
          <div
            class="LandingFeaturesText-topText-46"
          >
            <h5
              class="MuiTypography-root-5 MuiTypography-h5-21"
            >
              Create, send and track invoices in minutes.
            </h5>
            <br />
            <p
              class="MuiTypography-root-5 MuiTypography-body1-14"
            >
              Create customer profiles, save recurring items and invoice your customers from anywhere, so you can get paid faster!
            </p>
          </div>
        </div>
        <div
          class="LandingFeaturesText-bottom-47"
        >
          <div
            class="LandingFeaturesText-bottomText-48"
          >
            <h5
              class="MuiTypography-root-5 MuiTypography-h5-21"
            >
              Comprehensive dashboard provides all your data at a glance.
            </h5>
            <br />
            <p
              class="MuiTypography-root-5 MuiTypography-body1-14"
            >
              Gain insight into your best customers, including those who need a friendly nudge. Forecast your business cashflow so you can plan accordingly.
            </p>
          </div>
          <svg>
            undraw_finance_0bdk.svg
          </svg>
        </div>
      </div>
    </div>
    <div
      class="landing-get-started-container"
    >
      <div
        class="landing-get-started"
      >
        <div
          class="get-started-text"
        >
          STOP WAITING AND START INVOICING
        </div>
        <div
          class="card"
        >
          <div
            class="LandingGetStarted-root-49"
          >
            <div
              class="LandingGetStarted-section1-51"
            >
              <div
                class="MuiGrid-container-58 MuiGrid-align-items-xs-center-66"
              >
                <div
                  class="MuiGrid-item-59 MuiGrid-grid-xs-true-86"
                >
                  <h4
                    class="MuiTypography-root-5 MuiTypography-h4-20 MuiTypography-gutterBottom-32"
                  >
                    Free
                  </h4>
                </div>
              </div>
              <p
                class="MuiTypography-root-5 MuiTypography-body1-14 LandingGetStarted-features-55"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-155 LandingGetStarted-icon-54"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                  />
                </svg>
                Unlimited Invoices
              </p>
              <p
                class="MuiTypography-root-5 MuiTypography-body1-14 LandingGetStarted-lockedFeatures-56"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-155 LandingGetStarted-icon-54"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                  />
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                </svg>
                Unlimited Customers
              </p>
              <p
                class="MuiTypography-root-5 MuiTypography-body1-14 LandingGetStarted-lockedFeatures-56"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-155 LandingGetStarted-icon-54"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                  />
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                </svg>
                Unlimited Companies
              </p>
            </div>
            <hr
              class="MuiDivider-root-164 MuiDivider-middle-168"
            />
            <div
              class="LandingGetStarted-section2-52"
            >
              <p
                class="MuiTypography-root-5 MuiTypography-body1-14 MuiTypography-gutterBottom-32 LandingGetStarted-description-57"
              >
                Simply create a free account and get started with your free invoice on the go!
              </p>
            </div>
            <div
              class="LandingGetStarted-section3-53"
            >
              <button
                class="MuiButtonBase-root-195 MuiButton-root-169 MuiButton-contained-180 MuiButton-containedPrimary-181 MuiButton-raised-183 MuiButton-raisedPrimary-184 MuiButton-fullWidth-194 LandingGetStarted-greenButton-50"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label-170"
                >
                  Try For free
                </span>
                <span
                  class="MuiTouchRipple-root-198"
                />
              </button>
            </div>
          </div>
        </div>
        <div
          class="card"
        >
          <div
            class="LandingGetStarted-root-49"
          >
            <div
              class="LandingGetStarted-section1-51"
            >
              <div
                class="MuiGrid-container-58 MuiGrid-align-items-xs-center-66"
              >
                <div
                  class="MuiGrid-item-59 MuiGrid-grid-xs-true-86"
                >
                  <h4
                    class="MuiTypography-root-5 MuiTypography-h4-20 MuiTypography-gutterBottom-32"
                  >
                    Premium
                  </h4>
                </div>
                <div
                  class="MuiGrid-item-59"
                >
                  <h6
                    class="MuiTypography-root-5 MuiTypography-h6-22 MuiTypography-gutterBottom-32"
                  >
                    $6/month
                  </h6>
                </div>
              </div>
              <p
                class="MuiTypography-root-5 MuiTypography-body1-14 LandingGetStarted-features-55"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-155 LandingGetStarted-icon-54"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                  />
                </svg>
                Unlimited Invoices
              </p>
              <p
                class="MuiTypography-root-5 MuiTypography-body1-14 LandingGetStarted-features-55"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-155 LandingGetStarted-icon-54"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                  />
                </svg>
                Unlimited Customers
              </p>
              <p
                class="MuiTypography-root-5 MuiTypography-body1-14 LandingGetStarted-features-55"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-155 LandingGetStarted-icon-54"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                  />
                </svg>
                Unlimited Companies
              </p>
            </div>
            <hr
              class="MuiDivider-root-164 MuiDivider-middle-168"
            />
            <div
              class="LandingGetStarted-section2-52"
            >
              <p
                class="MuiTypography-root-5 MuiTypography-body1-14 MuiTypography-gutterBottom-32 LandingGetStarted-description-57"
              >
                Unlimited invoices for unlimited customers. Create an account to save your invoices, customers and company settings for convenient access.
              </p>
            </div>
            <div
              class="LandingGetStarted-section3-53"
            >
              <button
                class="MuiButtonBase-root-195 MuiButton-root-169 MuiButton-contained-180 MuiButton-containedPrimary-181 MuiButton-raised-183 MuiButton-raisedPrimary-184 MuiButton-fullWidth-194 LandingGetStarted-greenButton-50"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label-170"
                >
                  Get Premium
                </span>
                <span
                  class="MuiTouchRipple-root-198"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
